<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DesktopPanel</class>
 <widget class="QFrame" name="DesktopPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>206</width>
    <height>40</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="frameShape">
   <enum>QFrame::Panel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolBar" name="toolbar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <addaction name="action_pin"/>
     <addaction name="action_settings"/>
     <addaction name="action_monitors"/>
     <addaction name="action_autosize"/>
     <addaction name="action_fullscreen"/>
     <addaction name="action_autoscroll"/>
     <addaction name="action_cad"/>
     <addaction name="action_send_keys"/>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>5</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>5</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
  <action name="action_alt_tab">
   <property name="text">
    <string notr="true">Alt+Tab</string>
   </property>
  </action>
  <action name="action_alt_shift_tab">
   <property name="text">
    <string notr="true">Alt+Shift+Tab</string>
   </property>
  </action>
  <action name="action_printscreen">
   <property name="text">
    <string notr="true">PrintScreen</string>
   </property>
  </action>
  <action name="action_alt_printscreen">
   <property name="text">
    <string notr="true">Alt+PrintScreen</string>
   </property>
  </action>
  <action name="action_custom">
   <property name="text">
    <string>Custom...</string>
   </property>
  </action>
  <action name="action_pin">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icon/pin.png</normaloff>:/icon/pin.png</iconset>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="toolTip">
    <string>Lock toolbar</string>
   </property>
  </action>
  <action name="action_settings">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icon/settings.png</normaloff>:/icon/settings.png</iconset>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="toolTip">
    <string>Session configuration</string>
   </property>
  </action>
  <action name="action_monitors">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icon/monitors.png</normaloff>:/icon/monitors.png</iconset>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="toolTip">
    <string>Monitor selection</string>
   </property>
  </action>
  <action name="action_autosize">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icon/arrow-in-out.png</normaloff>:/icon/arrow-in-out.png</iconset>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="toolTip">
    <string>Automatic window size</string>
   </property>
  </action>
  <action name="action_fullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icon/application-resize-full.png</normaloff>:/icon/application-resize-full.png</iconset>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="toolTip">
    <string>Switch to full screen mode or back</string>
   </property>
  </action>
  <action name="action_autoscroll">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icon/auto-scroll.png</normaloff>:/icon/auto-scroll.png</iconset>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="toolTip">
    <string>Enabling/disabling automatic scrolling</string>
   </property>
  </action>
  <action name="action_cad">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icon/ctrl-alt-del.png</normaloff>:/icon/ctrl-alt-del.png</iconset>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="toolTip">
    <string>Send Ctrl+Alt+Delete</string>
   </property>
  </action>
  <action name="action_send_keys">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icon/keyboard.png</normaloff>:/icon/keyboard.png</iconset>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="toolTip">
    <string>Send key sequence</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
